exercises/phone-number/src/test/java/PhoneNumberTest.java

```diff
@@ -39,25 +39,15 @@ public void validWhen11DigitsAndFirstIs1() {
     }
 
     @Ignore
-    @Test
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
We've been migrating to using JUnit's ExpectedException @Rule feature for these cases (like in your PR #248), so if you could convert these two tests to use that, that would be great :)